components/gfx/display_list/mod.rs

```diff
@@ -139,8 +204,22 @@ impl DisplayList {
                                                                          point.y.to_f32_px()));
             Point2D::new(Au::from_f32_px(frac_point.x), Au::from_f32_px(frac_point.y))
         };
+    }
 
-        self.hit_test_contents(traversal, &translated_point, client_point, scroll_offsets, result);
+    #[inline]
+    fn scroll_root<'a>(scroll_root: &ScrollRoot,
+                       translated_point: &mut Point2D<Au>,
+                       scroll_offsets: &ScrollOffsetMap) {
+        // Adjust the translated point to account for the scroll offset if
+        // necessary. This can only happen when WebRender is in use.
```

Comment:
While you're here, could you remove the "This can only happen when WebRender is in use." bit? WebRender is always enabled now.